pyproject.toml

```diff
@@ -40,6 +40,7 @@ default_section = 'THIRDPARTY'
 force_grid_wrap = 0
 include_trailing_comma = true
 known_first_party = 'datadog_checks'
```

Comment:
note: `datadog_test_lib` can also be considered `known_first_party`, but this can be done in another PRs to reduce the changelog for this PR. The goal of this PR is mainly about upgrading isort version.